ui/app/src/components/AuditManagement/AuditManagement.tsx

```diff
@@ -132,7 +132,7 @@ export function AuditManagement(props: AuditManagementProps) {
       setDialogParams(parametersLookup[id]);
       auditLogEntryParametersDialogState.onOpen();
     } else {
-      fetchAuditLogEntry(id).subscribe({
+      fetchAuditLogEntry(id, site).subscribe({
```

Comment:
What about global?

Reply:
Site prop is optional (null in global), and the service validates siteId, if null, the param is not sent